cityofaustin/atd-moped
# Use new Moped vector tile endpoint

## Associated issues


https://github.com/cityofaustin/atd-data-tech/issues/8157


## Testing
**URL to test:** 

https://8157-new-tile-endpoint--atd-moped-main.netlify.app/moped/projects/383?tab=map

**Steps to test:**
Different things I did to test:
1. Add a signal. Save. Go back and edit signal
2. Add another point like feature. like a Refuge Island. 
3. Add a line like feature, like a highway or a bike lane. 
4. Add a hand drawn / user drawn feature, both  a point and a line. 
5. mix them up! see if I can delete one and not all. 

---
#### Ship list
